File: BookStore.APILayer/ClientApp/src/app/models/book.model.ts
import { DatePipe } from "@angular/common";
import { AuthorBookModel } from "./authorbook.model";
import { GenreBookModel } from "./genrebook.model";

export class BookModel {
    public id: number;
    public title: string;
    public authorBook: Array<AuthorBookModel>;
    public genreBook: Array<GenreBookModel>;
    public releaseDate: DatePipe;
    public price: number;
}

File: BookStore.BusinessLogicLayer/Models/BookModel.cs
using System;
using System.Collections.Generic;

namespace BookStore.BusinessLogicLayer.Models
{
    public class BookModel
    {
        public int ID { get; set; }
        public string Title { get; set; }
        public ICollection<AuthorBookModel> AuthorBook { get; set; }
        public ICollection<GenreBookModel> GenreBook { get; set; }
        public DateTime ReleaseDate { get; set; }
        public decimal Price { get; set; }
    }
}